artview/components/plot_radar.py

```diff
@@ -399,6 +388,10 @@ def NewTilt(self, variable, value, strong):
 
     def TiltSelectCmd(self, ntilt):
         '''Captures tilt selection and redraws the field with new tilt.'''
+        if ntilt < 0:
+            ntilt = len(self.rTilts)-1
+        elif ntilt >= len(self.rTilts):
+            ntilt = 0
         self.Vtilt.change(ntilt)
```

Comment:
This will and is happening when pressing down even if  tilt already equal 0. Note [this line](https://github.com/gamaanderson/artview/blob/pullrequest/artview/components/plot_radar.py#L127) has no test for that. The only reason we haven't notice this is because python is saving us by allowing negative index, now I'm just doing this looping around explicit, so I don't have to test Vtilt. Notice I have a [list](https://github.com/gamaanderson/artview/blob/pullrequest/DEV_README.txt#L88-L92) with valid values for the Vvariables, and Vtilt shouldn’t have negative values.